schemas/payments_schema.py

```
@@ -50,28 +52,31 @@ def set_id(cls,values):
 class PaymentBundles(BaseModel):
     bundleType:BundleType
     amount:int
-    numberOfstars:int
+    numberOfstars:Optional[int]=None
```

Ruff (0.14.10)

`Optional` may be undefined, or defined from star imports

(F405)